tests/gcp/operators/test_cloud_memorystore_system.py

```diff
@@ -18,31 +18,35 @@
 # under the License.
 """System tests for Google Cloud Memorystore operators"""
 
-from tests.gcp.operators.test_cloud_memorystore_system_helper import GCPCloudMemorystoreTestHelper
+import pytest
+
+from airflow.gcp.example_dags.example_cloud_memorystore import BUCKET_NAME
 from tests.gcp.utils.gcp_authenticator import GCP_MEMORYSTORE  # TODO: Update it
-from tests.test_utils.gcp_system_helpers import GCP_DAG_FOLDER, provide_gcp_context, skip_gcp_system
-from tests.test_utils.system_tests_class import SystemTest
+from tests.test_utils.gcp_system_helpers import GCP_DAG_FOLDER, GcpSystemTest, provide_gcp_context
+
+command = GcpSystemTest.commands_registry()
+
+
+@command
+def create_bucket():
+    GcpSystemTest.create_gcs_bucket(BUCKET_NAME, location="europe-north1")
 
 
-@skip_gcp_system(GCP_MEMORYSTORE, require_local_executor=True)
-class CloudBuildExampleDagsSystemTest(SystemTest):
-    """
-    System tests for Google Cloud Memorystore operators
+@command
+def delete_bucket():
+    GcpSystemTest.delete_gcs_bucket(BUCKET_NAME)
 
-    It use a real service.
-    """
-    helper = GCPCloudMemorystoreTestHelper()
 
-    @provide_gcp_context(GCP_MEMORYSTORE)
-    def setUp(self):
-        super().setUp()
-        self.helper.create_bucket()
+@pytest.fixture
+def helper():
+    create_bucket()
+    yield
+    delete_bucket()
 
-    @provide_gcp_context(GCP_MEMORYSTORE)
-    def test_run_example_dag(self):
-        self.run_dag('gcp_cloud_memorystore', GCP_DAG_FOLDER)
 
-    @provide_gcp_context(GCP_MEMORYSTORE)
-    def tearDown(self):
-        self.helper.delete_bucket()
-        super().tearDown()
+@command
+@GcpSystemTest.skip(GCP_MEMORYSTORE)
+@pytest.mark.usefixtures("helper")
+def test_run_example_dag():
```

Comment:
I think we should wait with full migration on the pytest syntax until we abandon the development of the 1.10.x line.  Now there may be low awareness about how to write tests using pytest, so we cannot require knowledge of it. Such decision-making should be made at the level of the entire project and we should send links and materials that will allow for easy familiarization. pytest is not part of the standard library, so not everyone knows it.
https://lists.apache.org/thread.html/08b64d3b084c865399f98f6c6f56235ce5329e843d97938e1a8045a5%40%3Cdev.airflow.apache.org%3Ehttps://lists.apache.org/thread.html/08b64d3b084c865399f98f6c6f56235ce5329e843d97938e1a8045a5%40%3Cdev.airflow.apache.org%3E

Reply:
Why wait for 1.10.x? We already moved to pytest in 1.10 as well (I cherry-picked all the changes there and 1.10.7 already uses pytest). 

We also updated 1.10.7 documentation with information that we use pytest and that we should (for example) prefer standard python asserts which are overwritten by pytest. 

I don't think abandoning 1.10 line has anything to do with pytest. 

We already as community agreed to go pytest-first (there was vote about it).

Reply:
Here is the thread where community voted on pytest - with few "Huge +1"s, "Big +1"s , "Awesomes", "Fantastic", "Love pytest", "Can't waits" and not a single negative reaction... 

https://lists.apache.org/thread.html/484add15d555c717bab82553ba7e43a4e04f0d403ccf954406b382e7%40%3Cdev.airflow.apache.org%3E

Reply:
I think this discussion should be on the mailing list.  In a moment I will create a new thread to discuss the topic of using new pytest features. Many people expressed their support for the question but did not comment on how to migrate the old convention to the new one. We don't have to use all the pytest features because the old code still works fine.

Reply:
This change affects code that is rather not "common" and rather no one uses this. In my opinion fixtures are easier to read. I think that we already agreed that we do not touch old test but new ones can be pytest-like.

Reply:
Yep. I think that was the agreement with asserts (and it is even documented in https://github.com/apache/airflow/blob/master/TESTING.rst#writing-unit-tests). 

We have not discussed other features of pytest, but with universal acceptance of pytest I think it's just a matter of code review between committers if new tests are using some of pytest new features or not. And I think it would be pointless if after converting to pytest we should refrain from using some useful features. For example I wanted to (At the next opportunity) to start using parametrized from pytest- because it allows to re-run single tests from parametrized set of tests. It's super useful and we can have it in parallel to parameterized import. I wanted to try it out and if it works fine proposed to make it default.

I think this is a good idea when we start using new feature, agree (between reviewers) if it is better to use certain feature and before merging it just post a message to devlist that we are going to use it. I do not think we should have additional documentation/best practices etc. Pytes has already great documentation with many examples.

@mik-laj -> you did not explain why you thought we should wait for 1.10 abandoning? I think that was the proposal? What was the reasoning?

Reply:
Here is a thread on the mailing list https://lists.apache.org/thread.html/1e4df7d4b0cd9b2d2bb76a3336471aa85e852545dd41ada6d4e461b8%40%3Cdev.airflow.apache.org%3E